src/components/Navbar.jsx

```diff
@@ -1,17 +1,31 @@
 import React from 'react'
-import { AppBar, Box, Toolbar } from '@mui/material'
+import { AppBar, Box, TextField, Toolbar } from '@mui/material'
+
+
+export const Navbar = props => {
+
+    const insideMarginPx = 25
+    const calculatedWidth = props.containerWidth - insideMarginPx * 2
 
-export const Navbar = () => {
     return (
-            <AppBar position="static" sx={{ boxShadow: 'none', bgcolor: 'rgb(0 0 0 / 0%)' }}>
-                <Toolbar sx={{ display: 'flex', justifyContent: 'space-between', bgcolor: "white" }}>
-                    <Box
-                        component="img"
-                        sx={{ maxHeight: { xs: 25, md: 40 }, margin: '10px' }}
-                        alt="Silly Ashes logo. Mid green icon of ball hitting stumps next to 'Silly Ashes'."
-                        src="src/assets/sillashesLogo.png"
+        <AppBar position="sticky" sx={{ width: calculatedWidth, marginTop: '8px', borderRadius: '9px', maxHeight: '64px' }} >
+            <Toolbar sx={{ display: 'flex', justifyContent: 'space-between', bgcolor: "white", borderRadius: '8px' }}>
+                <Box
+                    component="img"
+                    sx={{ maxHeight: { xs: 25, md: 40 }, margin: '10px' }}
+                    alt="Silly Ashes logo. Mid green icon of ball hitting stumps next to 'Silly Ashes'."
```

Comment:
I love this style for each of the players boxes - but keep the original navbar.

Search box to be top right of navbar on lg viewports only..... otherwise not there
On smaller screens, the search bar should be full width as it's own thing underneath the nav bar